Magento_Theme/web/js/custom-script.js

```diff
@@ -0,0 +1,15 @@
+const noRouteParagraphParentRow = document.querySelector('.no-route-page-text');
+const noRouteParagraph = noRouteParagraphParentRow.querySelector('p');
+
+noRouteParagraph.innerHTML = noRouteParagraph.innerHTML.replace(
+  /(SPACE)/,
+  '<span class="no-route-space-word">$1</span>'
+);
+
+define(['jquery'], $ => {
+  return () => {
+    $('.no-route-page-title').click(function () {
```

Comment:
sometimes better to check if the element is really existing
1. add domReady!
2. check `if ($('.no-route-page-title').length) {}`